cpu/nrf5x_common/periph/uart.c

```diff
@@ -242,6 +276,7 @@ void uart_poweron(uart_t uart)
 
     if (isr_ctx[uart].rx_cb) {
         uart_config[uart].dev->TASKS_STARTRX = 1;
+        set_power(uart, false);
```

Comment:
```suggestion
        set_power(uart, true);
```

The UART is not activated otherwise.